torchtune/modules/optim/_cpu_offload.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+from torchtune.config._utils import _get_component_from_path
+
+try:
+    from torchao.prototype.low_bit_optim import (
+        CPUOffloadOptimizer as _CPUOffloadOptimizer,
+    )
+
+    _CPU_OFFLOAD_OPTIM_AVAILABLE = True
```

Comment:
I am not a big fan of adding this logic here only for the sake of the test. Would it be possible to handle this check in the test file?

Reply:
How do you propose to handle it? The annoying thing is torchao nightly doesn't bump the minor version (if you install torchao nightly now, the version will be `0.4.0+gitxxxx`). And there might be patch fix also (i.e. bump to `0.4.1`, which will not include CPU offload optimizer). In other words, I have to check for `0.4` only (don't check for bug fix) and check for `+gitxxx`). Feel kinda clunky for such a small thing.